### PR TITLE
Add roll option and aelike re support to hazards

### PR DESCRIPTION
**Pull request by CarlosFdez**
Closes https://github.com/foundryvtt/pf2e/issues/14444

image

### CODE REVIEW

**Comment by CarlosFdez** [comment]
This should be good to look at now.

File: src/module/actor/hazard/document.ts [hunk 1]
```diff
@@ -98,6 +98,14 @@ class HazardPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e |
         details.alliance = null;
     }
 
+    /** Run rule elements */
+    override prepareEmbeddedDocuments(): void {
+        super.prepareEmbeddedDocuments();
+        for (const rule of this.rules) {
```
Comment (stwlam) [inline src/module/actor/hazard/document.ts:104]: Why not just move this to the base class?

Comment (CarlosFdez) [reply src/module/actor/hazard/document.ts:104]: As a refactor? Sure. I'm down. I assumed there was a reason it wasn't.

Comment (stwlam) [reply src/module/actor/hazard/document.ts:104]: I think it was a redundant safeguard on them not getting executed by loot actors.

**Pull request merged**